src/main/scala/viper/gobra/ast/internal/utility/GobraStrategy.scala

```diff
@@ -128,7 +128,7 @@ object GobraStrategy {
       case (_: Capacity, Seq(exp: Expr)) => Capacity(exp)(meta)
       case (_: RangeSequence, Seq(low: Expr, high: Expr)) => RangeSequence(low, high)(meta)
       case (_: SequenceAppend, Seq(l: Expr, r: Expr)) => SequenceAppend(l, r)(meta)
-      case (_: SequenceUpdate, Seq(base: Expr, left: Expr, right: Expr)) => SequenceUpdate(base, left, right)(meta)
+      case (_: GhostCollectionUpdate, Seq(base: Expr, left: Expr, right: Expr)) => GhostCollectionUpdate(base, left, right)(meta)
```

Comment:
I think a lot of nodes are missing.

Reply:
For some reason, IntelliJ stopped warning me of inexhaustive matches. Thanks for noticing!

Reply:
Should be complete now